substrate/frame/staking/src/benchmarking.rs

```diff
@@ -1077,15 +1077,13 @@ mod tests {
 				(frame_benchmarking::BenchmarkParameter::v, v),
 				(frame_benchmarking::BenchmarkParameter::n, n),
 			];
-			let closure_to_benchmark =
```

Comment:
can this test be removed, it should be executed when running the benchmark tests anyway ?

Reply:
Yea looks a bit weird.